mlir/test/fusion/rock-gemm-align-tiling-multiple-args.mlir

```diff
@@ -1,8 +1,10 @@
-// RUN: rocmlir-driver --rock-gridwise-gemm-to-blockwise --rock-blockwise-gemm-to-threadwise --rock-linalg-align --verify-passes %s | rocmlir-opt
+// RUN: rocmlir-driver --rock-regularize %s | FileCheck %s
+// RUN: rocmlir-driver --rock-regularize --rock-gridwise-gemm-to-blockwise --rock-blockwise-gemm-to-threadwise --rock-linalg-align --verify-passes %s | rocmlir-opt
```

Comment:
Add a `FileCheck` with a `--check-prefix=` maybe?

Reply:
I think this is OK. We have several test cases with FileCheck to validate the expected structure of IR. This test aims to verify that the IR doesn't violate SSA.